tools-local/init-build.proj

```diff
@@ -7,7 +7,6 @@
   <Import Project="..\dir.props" />
 
   <UsingTask AssemblyFile="$(TasksBinDir)Microsoft.DotNet.SourceBuild.Tasks.dll" TaskName="GetHostInformation" />
-  <UsingTask AssemblyFile="$(TasksBinDir)Microsoft.DotNet.SourceBuild.Tasks.dll" TaskName="BuildDummyPackages" />
```

Comment:
I think this is similar to whether or not we should remove the reference assembly project: I don't think we should, in case this becomes a valid way to work around a problem again in the future.